medusa/notifiers/slack.py

```diff
@@ -91,8 +91,13 @@ def _send_slack(self, message=None, webhook=None):
             message = message.encode('utf-8')
 
         headers = {b'Content-Type': b'application/json'}
+        data=json.dumps(dict(
```

Comment:
it's still complaining about a missing whitespace.
but what you can do is, remove allot of the dict / json.dumps casting. It's not needed.
So just do:
```python
        data = {
            'text': message,
            'username': 'MedusaBot',
            'icon_url': 'https://cdn.pymedusa.com/images/ico/favicon-310.png'
        }
```

Reply:
Yeah, I was just sticking to what was already there. I'll make the change.